wp-content/plugins/jetpack/_inc/build/accessible-focus.min.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array(), 'version' => 'b3badd7020b82baf8fcc812448d43e22');
```

Comment:
:no_entry_sign: **Error( severity 5 )**: When a multi-item array uses associative keys, each value should start on a new line (*WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound*).